### PR TITLE
[IMP] add fr_CH translation

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

**Comment by tafaRU** [comment]
Technically :+1:

File: crm_lead_lost_reason/i18n/fr_CH.po [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: crm_lead_lost_reason
+#: model:ir.model,name:crm_lead_lost_reason.model_crm_lead_lost
+msgid " Ask a reason for the opportunity lost."
+msgstr " Demande un raison pour la perte de l'opportunité."
```
Comment (mdietrichc2c) [inline crm_lead_lost_reason/i18n/fr_CH.po]: s/un/une

File: crm_lead_lost_reason/i18n/fr_CH.po [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+#. module: crm_lead_lost_reason
+#: model:ir.actions.act_window,name:crm_lead_lost_reason.action_crm_lead_lost_reason
+#: model:ir.ui.menu,name:crm_lead_lost_reason.menu_crm_lead_lost_reason
+msgid "Lost Reasons"
+msgstr "Raisons de perte"
+
+#. module: crm_lead_lost_reason
+#: view:crm.lead:crm_lead_lost_reason.crm_case_form_view_oppor_reason
+msgid "Lost reason:"
+msgstr "Raison de la perte:"
```
Comment (gurneyalex) [inline crm_lead_lost_reason/i18n/fr_CH.po]: missing space before ':'

**Comment by gurneyalex** [comment]
please rename to fr.po there's nothing specifically swiss in there.

**Comment by damdam-s** [comment]
@gurneyalex @mdietrichc2c typos corrected and file renamed.
@all thanks

**Comment by mdietrichc2c** [comment]
:+1:

**Pull request merged**